src/transport/SecureSession.h

```diff
@@ -249,6 +252,11 @@ class SecureSession : public Session, public ReferenceCounted<SecureSession, Sec
         // grab this session, when all SessionHandles go out of scope, the
         // session object will be released automatically.
         kPendingRemoval = 3,
+
+        // For UpdateNOC command, the session still functional, but it can't
+        // yield any new exchanges, the last exchange running UpdateNOC command
+        // will close the session soon.
```

Comment:
```suggestion
        // The session is still functional, but it can't yield any new exchanges,
        // This is meant to be used in conjunction with 
        // ExchangeManager::AbortExchangesForFabricExceptOne, with the one
        // exceptional exchange handling moving this session out of this state when
        // it finishes whatever it needs the session for.
```